client/src/app/core/ui-services/base-filter-list.service.ts

```diff
@@ -40,7 +40,7 @@ export interface OsFilterOption {
  * and will receive their filtered data as observable
  */
 
-export abstract class BaseFilterListService<M extends BaseModel, V extends BaseViewModel> {
+export abstract class BaseFilterListService<V extends BaseViewModel> {
```

Comment:
The abstract BaseFilterListService does not need a BaseModel (anymore?). All extending classes have their baseModel removed, too.
Might need another check